acceptance/README.md

```diff
@@ -1,73 +1,97 @@
+1. Operate in a mannger and environment as similar to the production CI system as possible.
 
-#### Prepare the VM
+Unless otherwise indicated, all commands assume you are working in the top-level `puppet-server` directory
 
-No action is required here, VM's are already prepared and available in the vm
-pool.  For a list of all platforms prepared by the pool, please see the output
+## Acceptance Testing Using vmpooler
+
+VMs are already prepared and available in the VM pool. For a list of all platforms prepared by the pool, please see the output
 of:
 
     curl --url vmpooler.delivery.puppetlabs.net/vm
 
-#### Define a hosts config file for your new VM
-
-Copy/modify the local RHEL-7 host config at
-`acceptance/config/beaker/jenkins/redhat7-64m-64a.cfg`  This file configures a
-basic 2-node configuration.
+## Install dependencies
 
-Please note, the most recent host configurations can be generated with the
-`beaker-hostgenerator` command available from the `beaker-hostgenerator` gem.
-Generating a new host configuration takes the form of `bundle exec beaker-hostgenerator redhat7-64ma`
+Beaker needs to be installed and is expressed as a Gemfile dependency.  To
+install beaker, simply run `bundle install --path .bundle/gems` from the top
+level directory of the project.
 
-The host expression is a bit strange and is parsed as per the documentation at
-`bundle exec beaker-hostgenerator --help`.  See also [the
-source][beaker-hostgenerator].
+Running beaker will then be a matter of using `bundle exec beaker ...` as described
+below.
 
-[beaker-hostgenerator]: https://github.com/puppetlabs/beaker-hostgenerator
+You might also need to obtain the `~/.ssh/id\_rsa-acceptance` file from a
+colleague so that beaker can SSH into the target VMs located in the pooler.
 
-#### Define the PACKAGE_BUILD_VERSION and PUPPET_VERSION environment variables
+### Define your test environment using environment variables
 
-You'll need to provide a couple environment variables that specify which build
-of puppet-server to install and test against.
+You'll need to provide two environment variables that specify which builds of the
+agent and server to install and test against, ```PACKAGE_BUILD_VERSION``` and
+`PUPPET_VERSION`. ```PACKAGE_BUILD_VERSION``` indicates
+the build of the puppet-server package to install.
 
-1. Go to http://builds.delivery.puppetlabs.net/puppetserver/
-2. Scroll down to the most recent build at the bottom.  This will look like:
-   `0.1.4.SNAPSHOT.2014.05.15T1118`
-3. Copy the text (not the link address) - this will be `PACKAGE_BUILD_VERSION`
-4. Define ```PUPPET_VERSION``` as the packaged version of the puppet-agent
-   package that we're depending upon, which is currently `1.1.0`
+1. Go to [http://builds.puppetlabs.lan/puppetserver/?C=M;O=D](http://builds.puppetlabs.lan/puppetserver/?C=M;O=D).
+1. This puts the most recent build at the top of the display. It will look like `2016.1.1004.SNAPSHOT.2016.02.03T1410`.
```

Comment:
The snapshot format here is for pe-puppet-server.  For puppet-server, the snapshot format is like this:
2.2.2.master.SNAPSHOT.2016.02.04T0323